docs/modules/ROOT/pages/support.adoc

```diff
@@ -1,33 +1,38 @@
 = Support
-:url-slack-hazelcast: https://join.slack.com/t/hazelcastcommunity/shared_invite/zt-feevu2bl-AWtL_N~pc7fGvdO9iKJAww
 :url-zendesk-hazelcast: https://hazelcast.zendesk.com/
 
 Hazelcast Cloud has different support options depending on your requirements. Here are the support levels:
 
-== Community
-
-Community support is available for free tier users. 
-*Support Channel:* link:{url-slack-hazelcast}[Slack]
-
 == Standard
 
 Standard support is available for all users with non-free clusters.
+
 *Coverage:* 24 x 7 x 365
+
 *Support Channel:* link:{url-zendesk-hazelcast}[ZenDesk]
-*SLA:* First Response < 1 Business Day
+
+*SLA:*
```

Comment:
```suggestion
*Our Promise to Respond:*
```

SLA doesn't seem very friendly. If we would rather use SLA, we should write it in full for anyone who does not know what that means (Service-Level Agreement).